### PR TITLE
[camera] Apply texture API change

### PR DESCRIPTION
**Pull request by bwikbs**
Related to https://github.com/flutter-tizen/engine/pull/259

### CODE REVIEW

**Comment by HakkyuKim** [comment]
Integration test CI is fixed. I'll rerun the test.

**Comment by bbrto21** [comment]
This PR should be rebased upon #344. because the example is dependent on the video plug-in.
but PR works fine.

File: packages/camera/tizen/src/camera_device.cc [hunk 1]
```diff
@@ -343,41 +343,26 @@ CameraDevice::CameraDevice(flutter::PluginRegistrar *registrar,
...
-            if (prepared_packet_ && !current_packet_) {
-              current_packet_ = prepared_packet_;
-              prepared_packet_ = nullptr;
-            }
-
             tbm_surface_h surface = nullptr;
             int ret = media_packet_get_tbm_surface(current_packet_, &surface);
             if (ret != MEDIA_PACKET_ERROR_NONE) {
               LOG_ERROR("media_packet_get_tbm_surface failed, error: %d", ret);
-              media_packet_destroy(current_packet_);
```
Comment (bbrto21) [inline packages/camera/tizen/src/camera_device.cc:366]: Why do you remove this line? if getting a tbm surface from media packets is fails, shouldn't media packets be destroyed? The reference is lost on the very next line.

File: packages/camera/tizen/src/camera_device.cc [hunk 2]
```diff
@@ -343,41 +343,26 @@ CameraDevice::CameraDevice(flutter::PluginRegistrar *registrar,
           [this](size_t width,
                  size_t height) -> const FlutterDesktopGpuBuffer * {
             std::lock_guard<std::mutex> lock(mutex_);
-            if (prepared_packet_ && !IsValidMediaPacket(prepared_packet_)) {
-              media_packet_destroy(prepared_packet_);
-              prepared_packet_ = nullptr;
-            }
-            if (current_packet_ && !IsValidMediaPacket(current_packet_)) {
```
Comment (bbrto21) [inline packages/camera/tizen/src/camera_device.cc:350]: Could you remove `IsValidMediaPacket` definition too.

Comment (bwikbs) [reply packages/camera/tizen/src/camera_device.cc:350]: @bbrto21 Oh. good point!

File: packages/camera/tizen/src/camera_device.cc (hunk 1 above)
Comment (bwikbs) [reply packages/camera/tizen/src/camera_device.cc:366]: You're right. It seems to be a mistake! 👍

**Review by bbrto21 — APPROVED** [review]

**Pull request merged**